providers/runtime.go

```diff
@@ -588,11 +589,12 @@ func (r *Runtime) lookupResourceProvider(resource string) (*ConnectedProvider, *
 		// version support, we can safely remove this.
 		"go.mondoo.com/cnquery/v9/providers/core",
 		"go.mondoo.com/cnquery/v9/providers/network",
+		"go.mondoo.com/cnquery/v9/providers/os",
```

Comment:
I think we need to review those resources that require it from the k8s provider.

Reply:
it's just `containerImage` resource that is referenced